.prettierignore

```diff
@@ -1,4 +1,4 @@
-.github/
-*.config.js
-.prettierrc.js
-*.md
\ No newline at end of file
+# .github/
```

Comment:
Why are these ignored now?

Reply:
Ah, I was testing something out and didn't uncomment them. I will do so on my next PR.

Reply:
Fixed in latest commit.